book/installation.rst

```diff
@@ -98,6 +98,17 @@ number as the second argument of the ``new`` command:
     # Windows
     c:\projects\> php symfony.phar new my_project_name 2.3.23
 
+If you want your project to be based on the last Symfony LTS version, pass
```

Comment:
IIRC what my English teachers told me, ``last`` would mean that there won't be any other LTS version after this one, which is wrong.
But I will let a native speaker confirm this